test/multiverse/suites/grpc/Envfile

```diff
@@ -1,10 +1,13 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
 # frozen_string_literal: true
 
 suite_condition('grpc gem requires CRuby 2.5+') do
```

Comment:
We may also want to add something to this suite_condition about it not working on jRuby

Reply:
Addressed with 0d27cea76b43ab47a187df01c752db909c5416a1